src/components/common/FormField.tsx

```diff
@@ -47,6 +47,7 @@ const FormField: React.FC<{
   readOnly?: boolean;
   disabled?: boolean;
   numeric?: boolean;
+  whenValid?: (boolean) => void;
```

Comment:
if we're going to put another prop on `FormField`, i'd like to follow some naming convention - would `onValidate` make sense here? whatever you choose will need corresponding changes in `UserFormPage` as well

Reply:
also how does this differ from `isValidated` and `validators`? is it necessary to have all 3 props?

cc @aaronabraham311 - i think he was the one who added these originally

Reply:
+1 to changing to `onValidate`. Common practice is to name event handlers with `onSomething`, while the function you pass in would be `handleSomething`.